### PR TITLE
Event create

### PR DESCRIPTION
**Pull request by MaksimMatsveyenka**

### CODE REVIEW

File: frontend/src/components/AddEventComponent/AddEventComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    public render(): JSX.Element {
+        return (
+            <div>
+                {this.props.user &&
+                    <EventForm
+                        userId={this.props.user && this.props.user.id}
+                        config={SettingFormType.AddEvent}
+                        model={initFormForAddingNewEvent}
+                        submit={(payload: { event: Event, userId: number }) => this.props.addEvent({ event: payload.event, userId: payload.userId })}
```
Comment (dmitryRomankov) [inline frontend/src/components/AddEventComponent/AddEventComponent.tsx]: refactoring?

File: frontend/src/components/AddEventComponent/AddEventComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    }
+}
+
+const mapStateToProps = (state: AppState) => ({
+    authStatus: state.auth.status,
+    user: state.auth.user
+});
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+    addEvent: (payload: { event: Event, userId: number }) => dispatch(new AddEvent({ event: payload.event, userId: payload.userId })),
```
Comment (dmitryRomankov) [inline frontend/src/components/AddEventComponent/AddEventComponent.tsx]: same

File: frontend/src/components/AddEventComponent/AddEventComponent.tsx (hunk 1 above)
Comment (AuchynnikauViachaslau) [inline frontend/src/components/AddEventComponent/AddEventComponent.tsx]: Format this row

File: frontend/src/components/EventDescription/EventDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  CODE: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
+    fontFamily: '"Inconsolata", "Menlo", "Consolas", monospace',
+    fontSize: 16,
+    padding: 2,
+  },
+};
+
+class StyleButton extends React.Component<EventDescriptionProps, {}> {
+  public onToggle: (event: React.MouseEvent<HTMLSpanElement>) => void = (event: React.MouseEvent<HTMLSpanElement>) => {
```
Comment (dmitryRomankov) [inline frontend/src/components/EventDescription/EventDescription.tsx]: bad readability

File: frontend/src/components/EventForm/EventForm.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.ca-game-form {
+    &__container {
+        width: 144rem;
+        margin: 0 auto;
+        padding: 4rem;
```
Comment (dmitryRomankov) [inline frontend/src/components/EventForm/EventForm.scss]: padding in rem

File: frontend/src/components/EventForm/EventForm.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        background-color: white;
+        font-size:  1.6rem;
+    }
+
+    &__error {
+        color: #f92a42;
+        display: block;
+        padding: 5px 10px;
```
Comment (dmitryRomankov) [inline frontend/src/components/EventForm/EventForm.scss]: here padding px

File: frontend/src/components/EventForm/EventForm.tsx [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+      titleErrors = this.removeElFromArrByValue(
+        titleErrors,
+        frontEndValidationEventRegister.title.required
+      );
+    }
+
+    if (this.state.title.length < MIN_LENGTH_TITLE || this.state.title.length > MAX_LENGTH_TITLE) {
+      titleErrors.push(frontEndValidationEventRegister.title.length);
+    } else {
+      titleErrors = this.removeElFromArrByValue(titleErrors, frontEndValidationEventRegister.title.length);
```
Comment (dmitryRomankov) [inline frontend/src/components/EventForm/EventForm.tsx]: look at line 105

File: frontend/src/components/EventForm/EventForm.tsx [hunk 7]
```diff
@@ -0,0 +1,396 @@
...
+        frontEndValidationEventRegister.address.length
+      );
+    }
+
+    if (this.state.address.length < MIN_LENGTH_ADDRESS || this.state.address.length > MAX_LENGTH_ADDRESS) {
+      addressErrors.push(frontEndValidationEventRegister.address.length);
+    } else {
+      addressErrors = this.removeElFromArrByValue(
+        addressErrors,
+        frontEndValidationEventRegister.address.length);
```
Comment (dmitryRomankov) [inline frontend/src/components/EventForm/EventForm.tsx]: ) -_-

File: frontend/src/components/EventForm/EventForm.tsx (hunk 6 above)
Comment (MaksimMatsveyenka) [reply frontend/src/components/EventForm/EventForm.tsx]: ?

**Pull request merged**